yellowbrick/classifier/classification_report.py

```diff
@@ -191,7 +191,7 @@ def score(self, X, y):
         super(ClassificationReport, self).score(X, y)
 
         y_pred = self.predict(X)
-        scores = precision_recall_fscore_support(y, y_pred)
+        scores = precision_recall_fscore_support(y, y_pred, labels=self.classes_)
```

Comment:
This was quite an ingenious approach however @admo120104 your changes cause classification report tests to fail.  Now if you specify classes within the visualizer then it raises an error.

```TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''```

Here is the code that no longer works because of your changes
```
from sklearn.model_selection import TimeSeriesSplit
from sklearn.naive_bayes import GaussianNB

from yellowbrick.classifier import ClassificationReport
from yellowbrick.datasets import load_occupancy

# Load the classification dataset
X, y = load_occupancy()

# Specify the target classes
classes = ["unoccupied", "occupied"]

# Create the training and test data
tscv = TimeSeriesSplit()
for train_index, test_index in tscv.split(X):
    X_train, X_test = X.iloc[train_index], X.iloc[test_index]
    y_train, y_test = y.iloc[train_index], y.iloc[test_index]

# Instantiate the classification model and visualizer
model = GaussianNB()
visualizer = ClassificationReport(model, support=True, classes=classes)

visualizer.fit(X_train, y_train)        # Fit the visualizer and the model
visualizer.score(X_test, y_test)        # Evaluate the model on the test data
visualizer.show()                       # Finalize and show the figure
```

Reply:
The tests are failing on MacOS and Ubuntu

Reply:
@admo1 This code fixes the issues on MacOS

```
#labels must be a data type that works with np.isnan
labels = range(len(self.classes_))
scores = precision_recall_fscore_support(y, y_pred, labels=labels)
```

Reply:
@admo1 It has been really nice working with you.  Can you write a test for this issue?  Then I will close out the PR!